compiler/rustc_borrowck/src/borrowck_errors.rs

```diff
@@ -29,17 +30,12 @@ impl<'cx, 'tcx> crate::MirBorrowckCtxt<'cx, 'tcx> {
         borrow_span: Span,
         borrow_desc: &str,
     ) -> DiagnosticBuilder<'tcx, ErrorGuaranteed> {
-        let mut err = struct_span_err!(
-            self,
+        self.infcx.tcx.sess.create_err(CannotUseWhenMutablyBorrowed {
             span,
-            E0503,
-            "cannot use {} because it was mutably borrowed",
+            borrow_span,
+            borrow_desc,
             desc,
-        );
-
-        err.span_label(borrow_span, format!("{} is borrowed here", borrow_desc));
-        err.span_label(span, format!("use of borrowed {}", borrow_desc));
```

Comment:
These labels weren't added to the ftl file, where did they go?

Reply:
Should they be added as separate messages with their labels something like this
```
borrowck_cannot_use_when_mutably_borrowed = {$borrow_span} is borrowed here
```

Reply:
You should preserve all the error messages as much as possible